includes/admin/class-wc-stripe-settings-controller.php

```diff
@@ -102,7 +102,6 @@ public function admin_scripts() {
 			),
 			'is_upe_checkout_enabled' => WC_Stripe_Feature_Flags::is_upe_checkout_enabled(),
 			'stripe_oauth_url'        => $oauth_url,
-			'accountStatus'           => $this->account->get_account_status(),
```

Comment:
No longer needed, we're now just getting the data from the `useAccount()` hook in `wcstripe/data`